_docs/mobile/android.md

```diff
@@ -8,7 +8,7 @@ order: 200
 
 Get the DuckDuckGo Private Browser from [Google Play](https://play.google.com/store/apps/details?id=com.duckduckgo.mobile.android), [F-Droid](https://f-droid.org/packages/com.duckduckgo.mobile.android/), or [as an APK File](https://github.com/duckduckgo/Android/releases).
 
-Our free web browser for {% include ddg-browser-os-list-linked.md %} is a privacy-protecting alternative to Google Chrome and Search. {% include browser-features.md %}
+Our free web browser is a privacy-protecting alternative to Google Chrome and other browsers. It comes with seamless privacy protection built-in, including our search engine that replaces Google and doesn’t track your search history, plus {% include browser-benefits.md %}, and more. The DuckDuckGo browser is available for {% include ddg-browser-os-list-linked.md %}. 
```

Comment:
on the iOS page we say "Our free web browser **for iOS**"

Not sure if we intentionally don't do the same here?